config/main.py

```diff
@@ -69,6 +74,35 @@ def log_error(msg):
 # Helper functions
 #
 
+# Execute action on list of systemd services
+def execute_systemctl(list_of_services, action):
+    num_asic = _get_num_asic()
+    generated_services_list, generated_multi_instance_services = _get_sonic_generated_services()
+    if ((generated_services_list == []) and
+        (generated_multi_instance_services == [])):
+        log_error("Failed to get generated services")
+        return
+
+    for service in list_of_services:
+        if ((service + '.service' in generated_services_list) or
```

Comment:
it is more clear to have such logic

if service in generate_serivces_list:
   ...
else if service in multi_instance_service:
   if num_asic == 1:
   else:
else:
    log_warning("service xxx no in the list")

Reply:
Updated as per comment.